src/main/java/ru/sortix/parkourbeat/game/Game.java

```diff
@@ -209,4 +222,24 @@ public enum State {
         READY,
         RUNNING,
     }
-}
+
+    private void updateBossBar() {
+        Bukkit.getScheduler().runTaskTimer(getPlugin(), () -> {
```

Comment:
Когда игра завершится, у нас BukkitTask не завершится. Она будет вечно тикать. Потенциально утечка памяти и ресурсов CPU

Reply:
Можно добавить эвент, чтобы BukkitTask завершался.

Reply:
Как вариант - да. Можно как раз добавлять BukkitTask в поле, и потом по какому-то событию метод `cancel` вызывать у него.
Конкретно что за событие - надо смотреть подробнее.

Reply:
Надо найти метод, который завершает игру, в нем и можно отменять BukkitTask

Reply:
Мы можем использовать метод cancel() у объекта BukkitTask, чтобы прекратить его выполнение.

Reply:
Как такой вариант?

Reply:
Норм, но надо добавить ещё
```suggestion
        bossBarTask = Bukkit.getScheduler().runTaskTimer(getPlugin(), () -> {
```